src/MultiReport/SurfaceDial.cpp

```diff
@@ -50,25 +50,25 @@ static const uint8_t _hidMultiReportDescriptorSurfaceDial[] PROGMEM = {
     0x16, 0xf0, 0xf1, //     LOGICAL_MINIMUM (-3600)
     0x26, 0x10, 0x0e, //     LOGICAL_MAXIMUM (3600)
     0x81, 0x06,       //     INPUT (Data,Var,Rel)
-    // 0x09, 0x30,       //     USAGE (X)
-    // 0x75, 0x10,       //     REPORT_SIZE (16)
-    // 0x55, 0x0d,       //     UNIT_EXPONENT (-3)
-    // 0x65, 0x13,       //     UNIT (Inch,EngLinear)
-    // 0x35, 0x00,       //     PHYSICAL_MINIMUM (0)
-    // 0x46, 0xc0, 0x5d, //     PHYSICAL_MAXIMUM (24000)
-    // 0x15, 0x00,       //     LOGICAL_MINIMUM (0)
-    // 0x26, 0xff, 0x7f, //     LOGICAL_MAXIMUM (32767)
-    // 0x81, 0x02,       //     INPUT (Data,Var,Abs)
-    // 0x09, 0x31,       //     USAGE (Y)
-    // 0x46, 0xb0, 0x36, //     PHYSICAL_MAXIMUM (14000)
-    // 0x81, 0x02,       //     INPUT (Data,Var,Abs)
-    // 0x05, 0x0d,       //     USAGE_PAGE (Digitizers)
-    // 0x09, 0x48,       //     USAGE (Width)
-    // 0x36, 0xb8, 0x0b, //     PHYSICAL_MINIMUM (3000)
-    // 0x46, 0xb8, 0x0b, //     PHYSICAL_MAXIMUM (3000)
-    // 0x16, 0xb8, 0x0b, //     LOGICAL_MINIMUM (3000)
-    // 0x26, 0xb8, 0x0b, //     LOGICAL_MAXIMUM (3000)
-    // 0x81, 0x03        //     INPUT (Cnst,Var,Abs)
+    0x09, 0x30,       //     USAGE (X)
+    0x75, 0x10,       //     REPORT_SIZE (16)
+    0x55, 0x0d,       //     UNIT_EXPONENT (-3)
+    0x65, 0x13,       //     UNIT (Inch,EngLinear)
+    0x35, 0x00,       //     PHYSICAL_MINIMUM (0)
+    0x46, 0xc0, 0x5d, //     PHYSICAL_MAXIMUM (24000)
+    0x15, 0x00,       //     LOGICAL_MINIMUM (0)
+    0x26, 0xff, 0x7f, //     LOGICAL_MAXIMUM (32767)
+    0x81, 0x02,       //     INPUT (Data,Var,Abs)
+    0x09, 0x31,       //     USAGE (Y)
+    0x46, 0xb0, 0x36, //     PHYSICAL_MAXIMUM (14000)
+    0x81, 0x02,       //     INPUT (Data,Var,Abs)
+    0x05, 0x0d,       //     USAGE_PAGE (Digitizers)
+    0x09, 0x48,       //     USAGE (Width)
```

Comment:
Windows expects a value for the width if you want to use the position reporting.

Reply:
but the width is constant, minimum value is 3000, maximum value is 3000

Reply:
ok